### PR TITLE
Bump version to 2.11.0 and update CHANGELOG

### PR DESCRIPTION
**Pull request by rexcsn**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,26 +10,61 @@ CHANGELOG
 - Add support for Centos 7 ARM.
 - Add support for FSx Lustre DataCompressionType feature.
 - Add validation to prevent using a `cluster_resource_bucket` that is in a different region than the cluster.
+- Make sure slurmd service is not enabled before finalize stage, which will prevent user from unintentionally making compute node available in post-install process.
```
Comment (mauri-melato) [inline CHANGELOG.md:13]: Can we reword it in something that makes more clear the enhancement in terms of user' experience of a functionality. If it's minor let's remove it.

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,26 +10,61 @@ CHANGELOG
...
+- Change to ssh_target_checker.sh syntax that makes the script compatible with pdsh.
```
Comment (mauri-melato) [inline CHANGELOG.md:14]: Also in this case instead of what we did, let's talk about the enhancement if it's the case.

File: CHANGELOG.md [hunk 3]
```diff
@@ -10,26 +10,61 @@ CHANGELOG
...
-  - Add new SlurmctldParameters, power_save_min_interval=30, so power actions will be processed every 30 seconds
-  - Specify instance GPU model as GRES GPU Type in gres.conf, instead of previous hardcoded value for all GPU, Type=tesla
-- Upgrade Arm Performance Libraries (APL) to version 21.0.0
 - Make `key_name` parameter optional to support cluster configurations without a key pair.
 - Remove support for Python versions < 3.6.
-- Remove dependency on `future` package and `__future__` module.
+  - Remove dependency on `future` package and `__future__` module.
 - Root volume size increased from 25GB to 35GB on all AMIs. Minimum root volume size is now 35GB.
-- Add sanity check to prevent cluster creation in non officially supported AWS regions
+- Add sanity check to prevent cluster creation in non officially supported AWS regions.
```
Comment (mauri-melato) [inline CHANGELOG.md:28]: Prevent cluster creation in an AWS region not officially supported by ParallelCluster

**Review by mauri-melato — COMMENTED** [review]
As a general comment, let's keep changelog and version bump in two different PRs.

File: CHANGELOG.md (hunk 1 above)
Comment (rexcsn) [reply CHANGELOG.md]: Removed, the change can be found in cookbook changelog

File: CHANGELOG.md (hunk 2 above)
Comment (rexcsn) [reply CHANGELOG.md]: Removed, the change can be found in cookbook changelog

File: CHANGELOG.md (hunk 3 above)
Comment (rexcsn) [reply CHANGELOG.md]: Done

File: CHANGELOG.md [hunk 4]
```diff
@@ -10,26 +10,61 @@ CHANGELOG
...
 - Change instance IAM policies to restrict run-instance API to only launching instances in the compute subnet.
+- Upgrade EFA installer to version 1.12.1.
+  - EFA configuration: ``efa-config-1.8`` (from ``efa-config-1.7``)
+  - EFA profile: ``efa-profile-1.5`` (from ``efa-profile-1.4``)
+  - EFA kernel module: ``efa-1.12.1`` (from ``efa-1.10.2``)
+  - RDMA core: ``rdma-core-32.1amzn`` (from ``rdma-core-31.2amzn``)
+  - Libfabric: ``libfabric-1.11.2amzon1.1`` (from ``libfabric-1.11.1amzn1.0``)
+  - Open MPI: ``openmpi40-aws-4.1.1`` (from ``openmpi40-aws-4.1.0``)
+- Increase timeout when attaching EBS volumes from 3 to 5 minutes.
+- Retry `berkshelf` installation up to 3 times.
```
[the remark this answers is not in the document]
Comment (rexcsn) [reply CHANGELOG.md]: Removed, the change can be found in cookbook changelog

**Review by mauri-melato — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 5]
```diff
@@ -10,26 +10,61 @@ CHANGELOG
...
-- Upgrade Arm Performance Libraries (APL) to version 21.0.0
 - Make `key_name` parameter optional to support cluster configurations without a key pair.
 - Remove support for Python versions < 3.6.
-- Remove dependency on `future` package and `__future__` module.
+  - Remove dependency on `future` package and `__future__` module.
 - Root volume size increased from 25GB to 35GB on all AMIs. Minimum root volume size is now 35GB.
-- Add sanity check to prevent cluster creation in non officially supported AWS regions
+- Add sanity check to prevent cluster creation in non officially supported AWS regions.
 - Change instance IAM policies to restrict run-instance API to only launching instances in the compute subnet.
+- Upgrade EFA installer to version 1.12.1.
```
[the remark this answers is not in the document]
Comment (mauri-melato) [reply CHANGELOG.md:28]: It's time to update it with EFA 1.12.2 info

File: CHANGELOG.md [hunk 6]
```diff
@@ -10,26 +10,57 @@ CHANGELOG
 - Add support for Centos 7 ARM.
```
Comment (mauri-melato) [inline CHANGELOG.md:10]: Remove

**Pull request merged**